Scuti/Scripts/UI/Store/OffersPresenterUniversal.cs

```diff
@@ -236,10 +244,11 @@ async protected override Task PopulateOffers(CancellationToken cancelToken)
             isInitilize = true;
         }
 
-		private void CalculateColumnWidth()
+		private void IntiColumnSystem()
 		{
-			columnSystem.ColumnWidth = widgetPrefab_Large.GetComponent<RectTransform>().rect.width;
-			columnSystem.Init();
+			var tempColumnWidth = widgetPrefab_Large.GetComponent<RectTransform>().rect.width;
```

Comment:
Curious if calling widgetPrefab_Large.transform and casting it is faster or not than GetComponent